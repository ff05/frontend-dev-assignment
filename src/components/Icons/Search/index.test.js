import React from 'react';
import IconsSearch from './index';
import { shallow } from 'enzyme';

describe('<IconsSearch />', () => {
  const renderedComponent = shallow(<IconsSearch />);

  it('renders correctly', () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
