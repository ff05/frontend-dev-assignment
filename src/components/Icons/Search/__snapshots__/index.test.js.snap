// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconsSearch /> renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IconsSearch
    isFocus={false}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <svg
        viewBox="0 0 451 451"
>
        <path
                d="M447.05 428l-109.6-109.6c29.4-33.8 47.2-77.9  47.2-126.1C384.65 86.2 298.35 0 192.35 0 86.25 0  .05 86.3.05 192.3s86.3 192.3 192.3 192.3c48.2 0  92.3-17.8 126.1-47.2L428.05 447c2.6 2.6 6.1 4 9.5  4s6.9-1.3 9.5-4c5.2-5.2 5.2-13.8 0-19zM26.95  192.3c0-91.2 74.2-165.3 165.3-165.3 91.2 0 165.3  74.2 165.3 165.3s-74.1 165.4-165.3 165.4c-91.1  0-165.3-74.2-165.3-165.4z"
        />
</svg>,
      "className": "IconsSearch",
      "type": "submit",
      "value": "Submit",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M447.05 428l-109.6-109.6c29.4-33.8 47.2-77.9  47.2-126.1C384.65 86.2 298.35 0 192.35 0 86.25 0  .05 86.3.05 192.3s86.3 192.3 192.3 192.3c48.2 0  92.3-17.8 126.1-47.2L428.05 447c2.6 2.6 6.1 4 9.5  4s6.9-1.3 9.5-4c5.2-5.2 5.2-13.8 0-19zM26.95  192.3c0-91.2 74.2-165.3 165.3-165.3 91.2 0 165.3  74.2 165.3 165.3s-74.1 165.4-165.3 165.4c-91.1  0-165.3-74.2-165.3-165.4z"
/>,
        "viewBox": "0 0 451 451",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M447.05 428l-109.6-109.6c29.4-33.8 47.2-77.9  47.2-126.1C384.65 86.2 298.35 0 192.35 0 86.25 0  .05 86.3.05 192.3s86.3 192.3 192.3 192.3c48.2 0  92.3-17.8 126.1-47.2L428.05 447c2.6 2.6 6.1 4 9.5  4s6.9-1.3 9.5-4c5.2-5.2 5.2-13.8 0-19zM26.95  192.3c0-91.2 74.2-165.3 165.3-165.3 91.2 0 165.3  74.2 165.3 165.3s-74.1 165.4-165.3 165.4c-91.1  0-165.3-74.2-165.3-165.4z",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <svg
          viewBox="0 0 451 451"
>
          <path
                    d="M447.05 428l-109.6-109.6c29.4-33.8 47.2-77.9  47.2-126.1C384.65 86.2 298.35 0 192.35 0 86.25 0  .05 86.3.05 192.3s86.3 192.3 192.3 192.3c48.2 0  92.3-17.8 126.1-47.2L428.05 447c2.6 2.6 6.1 4 9.5  4s6.9-1.3 9.5-4c5.2-5.2 5.2-13.8 0-19zM26.95  192.3c0-91.2 74.2-165.3 165.3-165.3 91.2 0 165.3  74.2 165.3 165.3s-74.1 165.4-165.3 165.4c-91.1  0-165.3-74.2-165.3-165.4z"
          />
</svg>,
        "className": "IconsSearch",
        "type": "submit",
        "value": "Submit",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <path
            d="M447.05 428l-109.6-109.6c29.4-33.8 47.2-77.9  47.2-126.1C384.65 86.2 298.35 0 192.35 0 86.25 0  .05 86.3.05 192.3s86.3 192.3 192.3 192.3c48.2 0  92.3-17.8 126.1-47.2L428.05 447c2.6 2.6 6.1 4 9.5  4s6.9-1.3 9.5-4c5.2-5.2 5.2-13.8 0-19zM26.95  192.3c0-91.2 74.2-165.3 165.3-165.3 91.2 0 165.3  74.2 165.3 165.3s-74.1 165.4-165.3 165.4c-91.1  0-165.3-74.2-165.3-165.4z"
/>,
          "viewBox": "0 0 451 451",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M447.05 428l-109.6-109.6c29.4-33.8 47.2-77.9  47.2-126.1C384.65 86.2 298.35 0 192.35 0 86.25 0  .05 86.3.05 192.3s86.3 192.3 192.3 192.3c48.2 0  92.3-17.8 126.1-47.2L428.05 447c2.6 2.6 6.1 4 9.5  4s6.9-1.3 9.5-4c5.2-5.2 5.2-13.8 0-19zM26.95  192.3c0-91.2 74.2-165.3 165.3-165.3 91.2 0 165.3  74.2 165.3 165.3s-74.1 165.4-165.3 165.4c-91.1  0-165.3-74.2-165.3-165.4z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        "type": "svg",
      },
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
